doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-coverage-report-dmn-based-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='test-scenarios-coverage-report-dmn-based-proc']
+= Coverage report for DMN-based test scenarios
+
+Review a coverage report for DMN-based test scenario by following the procedure below to validate your DMN models.
+
+.Prerequisites
+* The DMN-based test scenario template has been created for the selected test scenario.For more information about creating DMN-based test scenarios, see xref:test-designer-create-test-template-dmn-based-proc[].
+
+.Procedure
+. Open the DMN-based test scenario in the test scenario designer.
+. Click `Coverage Report` on the right of the test scenario designer to display the overview of the tests and the scenario status.
+
```

Comment:
I think we should add a note or similar to specify that decision coverage value consider all decisions successfully executed as covered regardless if there is any `EXPECT` condition on it nor the result of such `EXPECT` conditions (they can all fail from coverage perspective)

Reply:
@danielezonca, Can you please explain what is actually needed in that note?